cmd/osctl/cmd/config.go

```diff
@@ -144,8 +144,6 @@ func genV1Alpha1Config(args []string) {
 	}
 
 	input.AdditionalSubjectAltNames = additionalSANs
-	input.ControlPlaneEndpoint = canonicalControlplaneEndpoint
```

Comment:
do we need to set `input.ControlPlaneEndpoint` to anything?

Reply:
Gets set in the generation code for `v1alpha1`.